src/include/libpmem2.h

```diff
@@ -134,11 +136,14 @@ int pmem2_config_set_sharing(struct pmem2_config *cfg, unsigned type);
 int pmem2_config_set_protection(struct pmem2_config *cfg, unsigned flag);
 
 #define PMEM2_ADDRESS_ANY		0 /* default */
-#define PMEM2_ADDRESS_FIXED_REPLACE	1
+#define PMEM2_ADDRESS_FIXED_REPLACE	0 /* MAP_FIXED not supported yet */
```

Comment:
No, this is wrong - if someone would use PMEM2_ADDRESS_FIXED_REPLACE, pmem2_map will succeed with an address that doesn't match the requested one.